packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -1048,7 +1048,8 @@ onMounted(async () => {
 
 watch(
 	() => knobs.value,
-	async () => {
+	async (newValue, oldValue) => {
```

Comment:
the async is not necessary if their is no await.
```suggestion
(newValue, oldValue) => {
```